fineract-provider/src/main/java/org/apache/fineract/commands/service/AuditReadPlatformServiceImpl.java

```diff
@@ -164,7 +164,7 @@ public AuditData mapRow(final ResultSet rs, @SuppressWarnings("unused") final in
 
     @Override
     public Collection<AuditData> retrieveAuditEntries(final SQLBuilder extraCriteria, final boolean includeJson) {
-        return retrieveEntries("audit", extraCriteria, " order by aud.id DESC limit " + PaginationParameters.getCheckedLimit(null),
+        return retrieveEntries("audit", extraCriteria, " order by aud.id DESC ",
```

Comment:
This will cause performance issues if generated from frontend when the audit trail grows large . 
example is VFD when we started having timeouts on getting audit entries

Reply:
Its only for export not for page.

Reply:
still the same effect . I'll resolve this comment, but maybe you should confirm again from the client whhy they want this . It might not be usable in the future due to  how large data would have grown

Reply:
Done they want it for few months but current functionality only allowed for page entries which is only 15. So they want this.